app/views/propertyLinking/ownershipToProperty.scala.html

```diff
@@ -55,7 +55,7 @@
 
 @getEnrichedErrorLinks = @{
     toCustomDateFormErrors.toSet.map{ error: FormError =>
-        ErrorLink(href = Some(s"#${error.key}"),
+        ErrorLink(href = Some(s"#${error.key}.day"),
```

Comment:
why `day`? (what about month and year?)

Reply:
@darrell.plant I spoke to @will.blakeney and he said only Day field should be focused, as the error summary message is only one and doesn't break down into day, month and year.

Reply:
yep, I was just puzzled, but makes sense

Reply:
and before adding `day` it didn't work - got it now :)